Framework/Kernel/src/Matrix.cpp

```diff
@@ -975,36 +970,37 @@ T Matrix<T>::Invert()
   @return Determinant (0 if the matrix is singular)
 */
 {
-  if (nx != ny && nx < 1)
+  if (m_numRowsX != m_numColumnsY && m_numRowsX < 1)
     return 0;
 
-  if (nx == 1) {
-    T det = V[0][0];
-    if (V[0][0] != static_cast<T>(0.))
-      V[0][0] = static_cast<T>(1.) / V[0][0];
+  if (m_numRowsX == 1) {
+    T det = m_rawData[0][0];
+    if (m_rawData[0][0] != static_cast<T>(0.))
+      m_rawData[0][0] = static_cast<T>(1.) / m_rawData[0][0];
     return det;
   }
-  auto indx = new int[nx]; // Set in lubcmp
+  std::vector<int> indx;
```

Comment:
Bit of a brain fart when I was writing that - I completely forgot you can set the size on construction, there is another place in this file I'll also change that too.